user_config.c

```diff
@@ -0,0 +1,73 @@
+#include "user_config.h"
+#include <stdio.h>
+
+static uint8_t set_debug_level = 0, change_function = 0, temp_num = 0;
+static bool set_sensor_messages = false;
+static char save_type;
+
+// function takes in a message and parses it to see if max_debug_level() or allow_sensor_messages() needs to be changed
+void parse_usb_string(const char *input) {
+	uint8_t iterate = 0;
+	while (input[iterate] != '\0') {
+		switch (change_function) {
+			// Case 2: last check for if there has been a valid message sent
+			// --> if parse_usb_string has received a semicolon
+			case 2:
+				change_function = 0;
+				if (input[iterate] == ';') {
+					if (save_type == 'G')
+						set_debug_level = temp_num;
+					else if (save_type == 'S')
+						set_sensor_messages = temp_num;
+					else if (save_type == 'L') {
+						printf("Current Config: \n");
+						printf("\tMax debug level: %d\n", max_debug_level());
+						if (allow_sensor_messages())
+							printf("\tSensor messages are allowed!\n");
```

Comment:
Can you use whatever the print function we use to print over usb is in usb_app.c? I don't actually remember what it's called.

you may need to use `sprintf` to get the numbers in there first.